react-app/src/styles/fonts.scss

```diff
@@ -0,0 +1,61 @@
+// Fonts
```

Comment:
At some point in time we should edit this file to contain the fonts etc that we want, although we don't need to do it now

Reply:
already deleted in the next pr :D